pyconuk/settings.py

```diff
@@ -0,0 +1,118 @@
+"""
+Django settings for pyconuk project.
+
+Generated by 'django-admin startproject' using Django 1.9.6.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/1.9/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/1.9/ref/settings/
+"""
+
+import os
+
+# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/1.9/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = 'secret'
+
+# SECURITY WARNING: don't run with debug turned on in production!
+DEBUG = True
+
+
+# Application definition
+
+INSTALLED_APPS = [
+    # These two apps are required for Django to work properly, even though we
+    # don't use them directly.
+    'django.contrib.auth',
+    'django.contrib.contenttypes',
+
+    'django.contrib.staticfiles',
+    
+    'django_amber',
+    'markdown_deux',
+
+    'pyconuk',
+]
+
+MIDDLEWARE_CLASSES = [
+]
+
+ROOT_URLCONF = 'pyconuk.urls'
+
+TEMPLATES = [
+    {
+        'BACKEND': 'django.template.backends.django.DjangoTemplates',
+        'DIRS': [os.path.join(BASE_DIR, 'templates')],
+        'APP_DIRS': True,
+        'OPTIONS': {
+            'context_processors': [
+                'django.template.context_processors.debug',
+                'django.template.context_processors.request',
+                'django.contrib.auth.context_processors.auth',
+                'django.contrib.messages.context_processors.messages',
+            ],
+        },
+    },
+]
+
+WSGI_APPLICATION = 'pyconuk.wsgi.application'
+
+
+# Database
+# https://docs.djangoproject.com/en/1.9/ref/settings/#databases
+
+DATABASES = {
+    'default': {
+        'ENGINE': 'django.db.backends.sqlite3',
+        'NAME': os.path.join(BASE_DIR, 'db.sqlite3'),
+    }
+}
+
+
+# Internationalization
+# https://docs.djangoproject.com/en/1.9/topics/i18n/
+
+LANGUAGE_CODE = 'en-gb'
+
+TIME_ZONE = 'UTC'
+
+USE_I18N = True
+
+USE_L10N = True
+
+USE_TZ = True
```

Comment:
We will be adding talk slots to the data model, so I should think having `USE_TZ` switched on will be useful.

I'll remove the `USE_\w\d\d\w` settings though.